main_2024.py

```diff
@@ -110,8 +110,8 @@ def main() -> int:
     pathlib.Path(LOG_DIRECTORY_PATH).mkdir(exist_ok=True)
     pathlib.Path(f"{LOG_DIRECTORY_PATH}/{start_time}").mkdir()
 
-    result, main_logger = logger.Logger.create("main")
-    if result:
+    create_logger_result, main_logger = logger.Logger.create("main")
+    if create_logger_result:
         frame = inspect.currentframe()
         main_logger.info("main logger initialized", frame)
```

Comment:
sorry we decided to change how logging works as per https://discord.com/channels/776618956638388305/1128878366724988998/1257862843747926036 (we return False, None if logger creation fails), I'll update the documentation